hash/saturn.xml

```diff
@@ -101,9 +101,9 @@ license:CC0
 	  Angelique Special 2 (Japan) [T-7627G]                                                 | ########MISSING########
 	  Angelique Special 2 (Japan) (Premium Box) [T-7638G]                                   |  @@
 	  Another Memories (Japan) [T-38001G]                                                   | (amemorie)
-	  Aponashi Girls: Olympos (Japan) (Shokai Gentei Cell Ga Package A) [T-4304G]           | - are these the same disc as the B package?
-	  Aponashi Girls: Olympos (Japan) (Shokai Gentei Cell Ga Package B) [T-4306G]           | (olympos)
-	  Aponashi Girls: Olympos (Japan) (Shokai Gentei Cell Ga Package C) [T-4307G]           | - are these the same disc as the B package?
+	  Aponashi Girls: Olympos (Japan) (Shokai Gentei Cel Ga Package A) [T-4304G]           | - are these the same disc as the B package?
+	  Aponashi Girls: Olympos (Japan) (Shokai Gentei Cel Ga Package B) [T-4306G]           | (olympos)
+	  Aponashi Girls: Olympos (Japan) (Shokai Gentei Cel Ga Package C) [T-4307G]           | - are these the same disc as the B package?
```

Comment:
The prevailing opinion seems to be to render ti as “Shokai Gentei Cel-ga”.

Reply:
Yeah, I just knew "cell" was bad. Changed as suggested.